arch/arm/soc/st_stm32/stm32l0/Kconfig.defconfig.series

```diff
@@ -0,0 +1,32 @@
+# Kconfig - ST Microelectronics STM32F0 MCU line
+#
+# Copyright (c) 2017 RnDity Sp. z o.o.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_SERIES_STM32L0X
+
+source "arch/arm/soc/st_stm32/stm32l0/Kconfig.defconfig.stm32l0*"
+
+config SOC_SERIES
+	default stm32l0
+
+if GPIO_STM32
+
+config GPIO_STM32_PORTD
+	default y
+
+config GPIO_STM32_PORTF
+	default y
+
+endif # GPIO_STM32
+
+if I2C && (I2C_1 || I2C_2 || I2C_3)
```

Comment:
Let's focus on GPIO/PINMUX/UART for the initial port.
Other IPs, like I2C, SPI could be added one by one once this initial port is merged.
It will reduce the review/rework load for this PR and enable faster merge.